components/sts/org.wso2.carbon.identity.sts.mgt.ui/src/main/resources/web/identity-sts/index.jsp

```diff
@@ -82,15 +83,15 @@ try {
 		              </tr>
 		              <tr>
 	                      <td><fmt:message key='card.name'/></td>
-	                      <td><%=cardIssuer.getCardName()%></td>	                     
+	                      <td><%=Encode.forHtmlContent(cardIssuer.getCardName())%></td>
 	                  </tr>
 	                  <tr>
 	                      <td><fmt:message key='valid.period'/></td>
-	                      <td><%=cardIssuer.getValidPeriodInDays()%></td>	                     
+	                      <td><%=cardIssuer.getValidPeriodInDays()%></td>
```

Comment:
cardIssuer.getValidPeriodInDays() returns an integer, hence doesn't need to Encode.